docs/livemedia-creator.rst

```diff
@@ -68,46 +69,47 @@ This is usually a good idea when testing changes to the kickstart. lmc tries
 to monitor the logs for fatal errors, but may not catch everything.
 
 
-How ISO creation works
-----------------------
-
-There are 2 stages, the install stage which produces a disk or filesystem image
-as its output, and the boot media creation which uses the image as its input.
-Normally you would run both stages, but it is possible to stop after the
-install stage, by using ``--image-only``, or to skip the install stage and use
-a previously created disk image by passing ``--disk-image`` or ``--fs-image``
+Overview
+---------
+First, lmc boots the passed ISO image containing Anaconda in a virtual machine
+with qemu. This serves as a reproducable environment for creating the new image.
+This step can be skipped with the :ref:`--no-virt option <no-virt>` to use the
+system lmc is running on instead.
 
-When creating an iso qemu boots using the passed Anaconda installer iso
-and installs the system based on the kickstart. The ``%post`` section of the
-kickstart is used to customize the installed system in the same way that
-current spin-kickstarts do.
+The virtual machine (or host system if ``--no-virt`` is used) runs Anaconda
+with the Kickstart file provided to lmc by the ``--ks`` option to create a new
+Fedora installation in a filesystem image. The ``%post`` section of the Kickstart
+is used to customize the installed system. The filesystem image is created from
+scratch by Anaconda within the virtual machine; it is not based upon the ISO image
+passed to lmc with the ``--iso`` option.
 
 livemedia-creator monitors the install process for problems by watching the
 install logs. They are written to the current directory or to the base
 directory specified by the --logfile command. You can also monitor the install
-by using a vnc client. This is recommended when first modifying a kickstart,
+by using a vnc client. This is recommended when first modifying a Kickstart,
 since there are still places where Anaconda may get stuck without the log
 monitor catching it.
 
-The output from this process is a partitioned disk image. kpartx can be used
-to mount and examine it when there is a problem with the install. It can also
-be booted using kvm.
+If specified by ``part`` commands in the Kickstart file, the output from the Anaconda
```

Comment:
Output when using a VM is always a partitioned image, it doesn't matter what's in the kickstart. When using --no-virt it is an unpartitioned filesystem image.